app/src/main/java/com/example/groceriesmanager/Fragments/RecipeSearchFragment.java

```diff
@@ -52,6 +54,8 @@ public class RecipeSearchFragment extends Fragment {
     private static final String QUERY_FILTER_VEGAN = "vegan";
     private static final String QUERY_FILTER_VEGETARIAN = "vegetarian";
     private static final String QUERY_FILTER_GLUTEN_FREE = "gluten-free";
+    User currentUser;
+    List<String> dietFilters;
```

Comment:
I mentioned using a bitwise comparison in another comment.  Check out https://docs.oracle.com/javase/7/docs/api/java/util/EnumSet.html which is a good way to implement something like this.